lib/main_common.pm

```diff
@@ -1431,6 +1431,7 @@ sub load_extra_tests_y2uitest_cmd {
     loadtest 'yast2_cmd/yast_sysconfig';
     loadtest 'yast2_cmd/yast_keyboard';
     loadtest 'yast2_cmd/yast_nfs_server';
+    loadtest 'yast2_cmd/yast_firewall' if is_sle('12-SP2+');
```

Comment:
Please also verify if these test are meant to run in SLE 15 codestream

Reply:
fixed.